lib/screens/task_list_screen.dart

```diff
@@ -95,7 +95,7 @@ class _TaskListScreenState extends State<TaskListScreen> {
                 semanticLabel: "Completed Task",
               )
             : Icon(
-                Icons.circle_outlined,
```

Comment:
Why the change from circle_outlined to circle?

Reply:
Actually current version giving me error so I have used another library from flutter icon.

Reply:
What was the error you had? Some of the Icons are only supported with the latest flutter version, so try `flutter upgrade` on your device and check if the default flutter icons work. I would prefer using the built in icons, instead of adding another dependency on Eva Icons, here.